mga/create-network-node.go

```diff
@@ -0,0 +1,84 @@
+package mga
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/wopta/goworkspace/lib"
+	"github.com/wopta/goworkspace/models"
+)
+
+type CreateNetworkNodeRequest struct {
+	Code        string                  `json:"code"`
+	Type        string                  `json:"type"`
+	Role        string                  `json:"role"`
+	NetworkCode string                  `json:"networkCode"`
+	ManagerUid  string                  `json:"managerUid,omitempty"`
+	ParentUid   string                  `json:"parentUid,omitempty"`
+	Agent       *models.AgentNode       `json:"agent,omitempty"`
+	Agency      *models.AgencyNode      `json:"agency,omitempty"`
+	Broker      *models.AgencyNode      `json:"broker,omitempty"`
+	Partnership *models.PartnershipNode `json:"partnership,omitempty"`
+	Products    []models.Product        `json:"products,omitempty"`
+}
```

Comment:
if all the fields are in common with NetworkNode, shouldn't we accept one as request, considering that we map what we want in the createNetworkNode function?